rider-core/src/main/java/com/github/database/rider/core/replacers/DateTimeReplacer.java

```diff
@@ -13,7 +13,7 @@
 public class DateTimeReplacer implements Replacer {
 
     public static final String PLACEHOLDER_FORMAT = "[%s,%s]"; // [prefix:placeholderName]
-    private SimpleDateFormat format = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+    public static final SimpleDateFormat DBUNIT_DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
```

Comment:
SimpleDateFormat is not thread safe, can't be static.

Reply:
Yes but it is final, no one will read a different value, wht could be the issue?

Reply:
Several threads could use single instance via static constant causing data corruption. SimpleDateFormat is not stateless and thread safe.

Reply:
How the data corruption would happen in our case? Could you detail?

Reply:
Only in case of parallel tests execution. Random value may be returned due to state concurrent access. A sample reference from google: https://javarevisited.blogspot.com/2012/03/simpledateformat-in-java-is-not-thread.html?m=1

Reply:
Only in case of parallel tests execution. Random value may be returned due to state concurrent access. A sample reference from google: https://javarevisited.blogspot.com/2012/03/simpledateformat-in-java-is-not-thread.html?m=1

Reply:
Got It, we may have issues in format or parse call, even if the variable is immutable, thank you.

Reply:
What about now @xpinjection? https://github.com/database-rider/database-rider/pull/132/commits/3a53fcf01cef8fbba91d5a97b31822eaed132073

Reply:
Now it is safe, performance impact is not important at all in our context. :)